00-Getting_Started/ubuntu-deps.sh

```diff
@@ -16,3 +15,7 @@ sudo apt-get install curl wget
 curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
 chmod +x kubectl
 sudo mv kubectl /usr/local/bin/
+
+# Add user to libvirt group and log in to it so that the change is effective in the current shell
+sudo adduser `id -un` libvirt
+newgrp libvirt
```

Comment:
The `newgrp` command starts a new shell (e.g. bash) in that new group but does not return. I'm wondering if it's better to remove that line and add it instead in the documentation (the markdown file) for the user to type manually.

Otherwise, it would not work correctly if the user runs the following:
```
./ubuntu-deps.sh && start_minikube.sh
```

Reply:
Oops, I had moved this operation to the end of the script for that reason but I missed the case where user executes both scripts as you said. So yeah, I agree with you, this solution wouldn't work in that case.

I looked for other solutions but they all start a new shell, for instance `` exec su -l `id -un` ``. Therefore, I don't see any command that can cover the case you mentioned.

So, yeah, we could add this as an instruction into the dependencies documentation or as a troubleshooting when starting minikube. 

How do you prefer to proceed?

Reply:
Maybe we could test instead of trying to do something automatic. Something like this in `start_minikube.sh`:

```
if id -Gn|grep -qvw libvirt ; then
  echo "User not in group libvirt"
  echo "Suggestion: run 'newgrp libvirt'"
  exit 1
fi
```

Reply:
I agree, given that we did not find any automatic solution that covers all cases, I think we can at least ensure that minikube is not executed if user does not belong to group libvirt in the current shell. I updated the PR accordingly.